real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyConfigFile.java

```diff
@@ -32,6 +32,8 @@
     public static final boolean LOGGING = true;
     public static final String LOG_FILE_LOCATION = "logs";
     public static final String LOG_FILE_NAME = "sensors";
+    public static final double LATITUDE = 40.441670;
```

Comment:
@abhinavGirish let's name this something better like `INITIAL_POS_LAT` and `INITIAL_POS_LON`